# Request
float64 start_x
float64 start_y
float64 end_x
float64 end_y
---
# Response
nav_msgs/Path path
float64 path_length